src/main/java/com/google/sps/data/User.java

```diff
@@ -19,14 +19,28 @@
 /** Class representing a user. */
 public class User {
 
-    // Contains the entity id from Datastore.
-    private long id;
+    // String representation of the key from Datastore.
```

Comment:
I changed the comment, but left the variable name as "id". In datastore it's called "name" because I'm storing the id value as a string, but I thought "id" was a clearer variable name.